### ISSUE TITLE
Is there any maintainer for this repo?

### ISSUE THREAD

**Issue by MehTheHedgehog** [vrld/hump#119]
I have a few PR/improvements for this project but looking one other issues this repo was abandoned.
Is there any new maintainer for this?

**Comment by MehTheHedgehog** [comment]
Ok, I found the answer.
Current most up to date repo: https://github.com/HDictus/hump/tree/temp-master

Leaving it for other looking for it

**Comment by PartyLich** [comment]
Thanks for sharing that fork, but I can't help noticing it doesn't have issues turned on. Is there any indication that the owner of that fork intended to take over maintaining an open source project, or did they just fork it to fix their own blockers?

**Comment by HDictus** [comment]
Thanks for pointing that out PartyLich, I hadn't noticed issues were disabled. 
I meant to maintain the fork so we can all continue to benefit from improvements to hump.

**Comment by PartyLich** [comment]
Awesome news! 

Many of the love related libs I've come across (being new to the engine) seem to beg the question "is this dead, or just complete?"

I'm not sure github offers a sane means of migrating issues even if you have access to both repos. Would you prefer existing and still relevant issues to be reposted on the new repo?

**Comment by vrld** [comment]
@HDictus would you mind if we made this sort of official? Meaning: I link to your repo in the readme, so people that find this one know where to go

I meant to tie up some loose ends, e.g., adding a module for lazy loading, rework the timer module to allow chaining, etc. But let's face it, I haven't been active in gamedev for a long, long while now and it's not looking like that will change in the future.

**Comment by flamendless** [comment]
Probably also update the links in `awesome-love2d` to point to the new maintained repo 🤔

**Comment by PartyLich** [comment]
The issue that got me involved was documentation, so getting the readthedocs transferred or a new one set up would be another task (and i guess im softly volunteering to help if i can).

⁣--

**Comment by HDictus** [comment]
> @HDictus would you mind if we made this sort of official? Meaning: I link to your repo in the readme, so people that find this one know where to go
> 
> I meant to tie up some loose ends, e.g., adding a module for lazy loading, rework the timer module to allow chaining, etc. But let's face it, I haven't been active in gamedev for a long, long while now and it's not looking like that will change in the future.

yeah that would be great. I had already put a note in awesome-love2d, but if you're officially stepping away we can change all the links to point to the fork. 

Transferring the documentation would be good to get started on too, I don't know how that is done but I guess I should start by making an account.

**Comment by HDictus** [comment]
@vrld  I'm having difficulty figuring out how to migrate readthedocs.io from your repository to my fork. I think that first you would need to unlink this repository on readthedocs, then I can link my fork. Are you stillhappy transferring maintenance to my fork?
